spec/request/vet360/address_request_spec.rb

```diff
@@ -151,5 +156,63 @@
         expect(errors_for(response)).to include "address-pou - can't be blank"
       end
     end
+
+    context 'when effective_end_date is included' do
+      let(:address) do
+        build(:vet360_address,
+              vet360_id: user.vet360_id,
+              effective_end_date: Time.now.utc.iso8601)
+      end
+
+      before do
+        allow_any_instance_of(User).to receive(:icn).and_return('1234')
+        address.id = 42
+      end
+
+      it 'effective_end_date is ignored', aggregate_failures: true do
+        VCR.use_cassette('vet360/contact_information/put_address_ignore_eed', VCR::MATCH_EVERYTHING) do
+          # The cassette we're using does not include the effectiveEndDate in the body.
```

Comment:
Can this context setting be expressed in a `context` block, instead of inline commenting?  As well as the other ones in the other spec files below.

Reply:
I see you checked this, but I am still seeing all the inline comments in the request specs.  I am missing something?

Reply:
@hpjaj I think the diff didn't update... but I see the changes now after pushing again. I modified the `context` and `it` text to be more explicit about what's being tested. And I also left the comments in to provide additional details of the implementation. I know you don't like inline comments but I think they might be appropriate in this case. Let me know if the changes still aren't showing up.

Reply:
Yeah, IMO, those inline comments are a code smell.